hardware_layer/hardware_arduino/src/arduino_node.cpp

```diff
@@ -30,284 +29,579 @@
 
 #define analogPinPressureSensor A0
 
-const float c092 = 506.22;
-const float s092 = -2.65;
-const float c093 = 448.62;
-const float s093 = -2.92;
-const float c099 = 397.65;  // reference as their graph is at lowest
-const float s099 = -2.71;   // reference as their graph is at lowest
-const float c113 = 539.85;
-const float s113 = -3.38;
-const float c117 = 441.32;
-const float s117 = -3.03;
-const float c122 = 547.39;
-const float s122 = -2.93;
-const int neutral_buoyancy_offset = 0;
+//The below variables are named in such fashion: <pwm-value-range><Direction-in-which-thruster-will-move><the-intended-thruster> 
+//for ex: 'highForwardEast' means "the east thruster of the bot will move forward with a high pwm value"    
+
+
+#define highForwardEast 
+#define medForwardEast
+#define lowForwardEast
+
+#define highForwardWest
+#define medForwardWest
+#define lowForwardWest
+
+#define highBackwardEast
+#define medBackwardEast
+#define lowBackwardEast
+
+#define highBackwardWest
+#define medBackwardWest
+#define lowBackwardWest
+
+#define highUpNorth
+#define medUpNorth
+#define lowUpNorth
+
+#define highUpSouth
+#define medUpSouth
+#define lowUpSouth
+
+#define highDownNorth
+#define medDownNorth
+#define lowDownNorth
+
+#define highDownSouth
+#define medDownSouth
+#define lowDownSouth
+
+#define highLeftNorth
+#define medLeftNorth
+#define lowLeftNorth
+
+#define highLeftSouth
+#define medLeftSouth
+#define lowLeftSouth
+
+#define highRightNorth
+#define medRightNorth
+#define lowRightNorth
+
+#define highRightSouth
+#define medRightSouth
+#define lowRightSouth
+
+#define highAnticlockNorth
+#define medAnticlockNorth
+#define lowAnticlockNorth
+
+#define highAnticlockSouth
+#define medAnticlockSouth
+#define lowAnticlockSouth
+
+#define highClockNorth
+#define medClockNorth
+#define lowClockNorth
+
+#define highClockSouth
+#define medClockSouth
+#define lowClockSouth
+
+#define highAnticlockEast
+#define medAnticlockEast
+#define lowAnticlockEast
+
+#define highAnticlockWest
+#define medAnticlockWest
+#define lowAnticlockWest
+
+#define highClockEast
+#define medClockEast
+#define lowClockEast
+
+#define highClockWest
+#define medClockWest
+#define lowClockWest
 
 MS5837 sensor;
+int SPEED_MODE;
```

Comment:
Use enum instead of this. Check [here](http://en.cppreference.com/w/cpp/language/enum)